self-managed/src/license_switching_sample.py

```diff
@@ -0,0 +1,140 @@
+                        'UsageOperation': DestinationContext
+                    }
+                )
+    print('AWS License Manager - CreateLicenseConversionTask API response:')
+    pprint.pprint(response)
+    return response
+
+# Helper method to track license conversion task status
+def get_license_conversion_task_status(AccountId, LicenseConversionTaskId):
+    assume_role_client = get_client_using_assume_role(AccountId, default_region)
+    response = assume_role_client.get_license_conversion_task(
+        LicenseConversionTaskId = LicenseConversionTaskId,
+    )
+    print('AWS License Manager - GetLicenseConfiguration API response:')
+    pprint.pprint(response)
+    # Status message in the response provides more details on the Staus
+    if ((response['Status'] == 'SUCCEEDED') or (response['Status'] == 'FAILED')):
+        return True
+    else: 
+        return False
+
+# Sample function to convert Windows Server from BYOL to license included
+# for all eligible conversion types please visit: https://docs.aws.amazon.com/license-manager/latest/userguide/conversion-types.html 
+def convert_BYOL_to_license_included_for_all_resources():
+    lm_client = get_client(default_region)
+    
+    # Pass filters to apply conversion on certain resources
+    # https://docs.aws.amazon.com/license-manager/latest/APIReference/API_ListResourceInventory.html#licensemanager-ListResourceInventory-request-Filters 
+    response = lm_client.list_resource_inventory(
+            MaxResults=20,
+        )
+    
+    pending_conversion_tasks = []
+    while True:
+        print('AWS License Manager - ListResourceInventory API response:')
+        pprint.pprint(response)
+        if "ResourceInventoryList" in response:
+            for resource in response['ResourceInventoryList']:
+                resource_arn = resource['ResourceArn']
+                account_id = resource_arn.split(':')[4]
+                converstion_task_response = create_license_conversion_task (account_id, resource_arn, 'RunInstances:0800', 'RunInstances:0002')
+                pending_conversion_tasks.append((account_id, converstion_task_response['LicenseConversionTaskId']))
+                pprint.pprint(converstion_task_response)
+        if "NextToken" in response:
+            next_token = response['NextToken']
+            response = lm_client.list_resource_inventory(
+                NextToken=next_token,
+            )
+        else:
+            break
+    while len(pending_conversion_tasks) != 0: 
+        for task in pending_conversion_tasks:
+            task_completed = get_license_conversion_task_status(task[0], task[1])
+            if task_completed == True: 
+                pending_conversion_tasks.remove(task)
+    return
+    
+# Sample function to convert Windows Server from LI to BYOL
+def convert_LI_to_BYOL_for_all_resources():
+    lm_client = get_client(default_region)
+    response = lm_client.list_resource_inventory(
```

Comment:
Does list_resource inventory take a filter of billing code? If it does then can you add billing code of "'RunInstances:0002". It will make it far more easier to understand for people who are using your code as reference.